sdmetrics/reports/single_table/_properties/column_pair_trends.py

```diff
@@ -56,11 +56,9 @@ def _convert_datetime_columns_to_numeric(self, data, metadata):
                         data[column_name] = pd.to_datetime(
                             data[column_name], format=datetime_format
                         )
-
-                    data.loc[~pd.isna(data[column_name]), column_name] = pd.to_numeric(
-                        data[column_name]
-                    )
```

Comment:
warning happened because the dtype of the column was datetime, but we were trying to assign some values to an int64 dtype.